src/utils/build_importer.py

```diff
@@ -7,66 +10,95 @@
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.support import expected_conditions as EC
-from yaml import dump
+from pydantic_yaml import to_yaml_str
 
+MAXROLL_PLANNER_URL = "maxroll.gg/d4/planner"
+MAXROLL_BUILD_GUIDE_URL = "maxroll.gg/d4/build-guides/"
+
+COOKIE_ACCEPT_BUTTON = "ncmp__btn"
+BUILD_PLANNER_CLASS = "d4t-PlannerLink"
+BUILD_PLANNER_OVERVIEW_CLASS = "d4t-Paperdoll"
+DIABLO_CLASS_NAME_CLASS = "d4t-ClassSelect"
+PLANNER_LABEL_CLASS = "d4t-label"
+ITEM_SLOT_CLASS = "d4t-slot"
+TITLE_CLASS = "d4t-title"
+ITEM_TYPE_CLASS = "d4t-category-header"
+ITEM_LIBRARY_CLASS = "d4t-ItemLibrary"
+ACTIVE_ITEM_CLASS = "d4t-active"
+PARENT_XPATH = "./../.."
+ITEM_MODIFIERS_CLASS = "d4t-item-mods"
+ITEM_RARITY_CLASS = "d4t-item-options"
+ITEM_NAME_CLASS = "d4t-header-title"
 
 def import_build():
     Logger.info(f"Paste maxroll.gg build guide or planner build here ie https://maxroll.gg/d4/build-guides/minion-necromancer-guide")
     url = input()
-    if "maxroll.gg/d4/planner" not in url and "maxroll.gg/d4/build-guides/" not in url:
+    if MAXROLL_PLANNER_URL not in url and MAXROLL_BUILD_GUIDE_URL not in url:
         Logger.info(f"Invalid url, please use a d4 planner build on MaxRoll.gg")
         return
-    options = webdriver.ChromeOptions()
-    options.add_argument("--headless=new")
-    driver = webdriver.Chrome(options=options)
+    match IniConfigLoader().general.browser:
```

Comment:
When changing the type to enum, you can simplify this a lot. Also, no need for a catch all at the end since the config is validated first and would never reach this code with an invalid value